### PR TITLE
Coming-soon-update

### PR DESCRIPTION
**Pull request by Nathanael-Rayapin**
Coming Soon page with filename lowercase

### CODE REVIEW

File: src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.ts [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
-export class ComingSoonFullScreenComponent {}
+export class ComingSoonFullScreenComponent {
+  socialIcons: socialLists[] = socialIconsLists;
+
+  // Because of the lack of informations about this button, I put the code like this
+  monetaryIcons: socialLists[] = [
+    {
+      name: 'toDefine',
+      path: 'toDefine',
+      src: '../../../../assets/img/coming-soon/zion.svg',
```
Comment (etsraphael) [inline src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.ts]: TEA is the official name

File: src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.html [hunk 2]
```diff
@@ -1 +1,47 @@
...
+          <div class="container-socials mx-5" *ngFor="let icon of socialIcons">
+            <a [href]="icon.path" target="_blank">
+              <img class="w-100 h-100" [src]="icon.src" [alt]="icon.name" />
+            </a>
+          </div>
+        </div>
+        <!-- Monetary Gifts -->
+        <div class="monetary-gifts col-12 mb-5">
+          <h3 class="col-10 col-md-7">
+            OR, You can buy TEA or you can give us monetary gifts
```
Comment (etsraphael) [inline src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.html]: To delete. We can't give this currently

File: src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.ts [hunk 3]
```diff
@@ -1,8 +1,31 @@
...
+    },
+    {
+      name: 'toDefine',
+      path: 'toDefine',
+      src: '../../../../assets/img/coming-soon/gift.svg',
```
Comment (etsraphael) [inline src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.ts]: To delete too. No gift right now

File: src/app/core/data/social-media-list.ts [hunk 4]
```diff
@@ -1,28 +1,43 @@
...
+export const socialIconsLists: socialLists[] = [
+  {
+    name: 'discord',
+    path: 'https://discord.gg/Kpdh29U6',
+    src: '../../../assets/img/coming-soon/discord.svg',
+  },
+  {
+    name: 'github',
+    path: 'https://github.com/etsraphael/WiinsWebDapp',
+    src: '../../../assets/img/core/icon/github.svg',
```
Comment (etsraphael) [inline src/app/core/data/social-media-list.ts:40]: This link doesn't work due to the uppercase letter. 
All the files in ``src/assets/img/core/icon`` have the be update with a lowercase as a first letter

File: src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.ts (hunk 1 above)
Comment (Nathanael-Rayapin) [reply src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.ts]: filename updated

File: src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.html (hunk 2 above)
Comment (Nathanael-Rayapin) [reply src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.html]: Section deleted

File: src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.ts (hunk 3 above)
Comment (Nathanael-Rayapin) [reply src/app/core/component/coming-soon-full-screen/coming-soon-full-screen.component.ts]: Array deleted

File: src/app/core/data/social-media-list.ts (hunk 4 above)
Comment (Nathanael-Rayapin) [reply src/app/core/data/social-media-list.ts:40]: file already in uppercase letter as we can see on this preview.
I'll try something about this bug

**Pull request merged**